spring-cloud-data-core/src/main/java/org/springframework/cloud/data/core/dsl/StreamDslParser.java

```diff
@@ -255,32 +366,38 @@ private ChannelNode eatChannelReference(boolean tapAllowed) {
 		if (tapAllowed && firstToken.data.equalsIgnoreCase("tap")) {
 			if (tokens.peek(TokenKind.DOT)) {
 				if (channelScopeComponents.size() < 3) {
-					tokens.raiseException(firstToken.startPos, DSLMessage.TAP_NEEDS_THREE_COMPONENTS);
+					tokens.raiseException(firstToken.startPos,
+							DSLMessage.TAP_NEEDS_THREE_COMPONENTS);
 				}
 				String tokenData = channelScopeComponents.get(1).data;
 				// for Stream, tap:stream:XXX - the channel name is always indexed
 				// for Job, tap:job:XXX - the channel name can have "." in case of job notification channels
 				if (!tokenData.equalsIgnoreCase("stream") && !tokenData.equalsIgnoreCase("job")) {
```

Comment:
Is there any reason why we use `job` instead of `task`? This question applies to all the references to the parser code including the DSL error messages.